receiver/elasticsearchreceiver/documentation.md

```diff
@@ -45,6 +45,8 @@ These are the metrics available for this scraper.
 | **elasticsearch.node.ingest.operations.failed** | Total number of failed ingest operations during the lifetime of this node. | {operation} | Sum(Int) | <ul> </ul> |
 | **elasticsearch.node.open_files** | The number of open file descriptors held by the node. | {files} | Sum(Int) | <ul> </ul> |
 | **elasticsearch.node.operations.completed** | The number of operations completed by a node. | {operations} | Sum(Int) | <ul> <li>operation</li> </ul> |
```

Comment:
Can't we add another attribute `result` to the existing metric through a feature gate? If user want the previous behavior then can aggregate the metric. For now aggregation can be done with `metricstransform` processor only, but it'll be available right in user setting of each scraping receiver soon (see for more details on that https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6429)

Even if we still want to add the new metrics in the time being, I believe they must be optional because they essentially represent the same data just broken down with another attribute.

Reply:
I'm not convinced. This attribute makes sense for `GET` operations, but not necessarily for other operations. `search` operations are already split on `query`, `fetch`, `scroll` and `suggest` - and none of these operations have something that would suggest a result.`merge`, `refresh`, `flush` and `warmer` operations only differentiate between `total` and `current` (or `periodic` in case of `flush`). In fact, only `index` operation also has a separate count for failures. 

I don't know what is the policy for such attributes - in most cases it will send redundant information, so I don't think it should be here. On the other hand, making these new metrics optional seems like a good idea for the reason you mentioned.

Reply:
It makes sense to me now. I agree it should be another metric, but probably disabled by default. Can you apply that change?

Reply:
Done.